### ISSUE TITLE
Run-on sentences make tutorial hard to read

### ISSUE THREAD

**Issue by hannahburkhardt** [googlecodelabs/android-testing#71]
Not an issue with the code but I don't know where else to put this. There are a lot of run-on sentences in the tutorial text that make it hard to follow the tutorial at times.
Examples:

Our repository here is only a mock object (and doesn't have any actual functionality), we need to fake the callback on the listener ourselves.

By default, all calls to the provided Android.jar throw exceptions, this allows you to write tests that do not depend on any Android platform behavior.

**Comment by dmeng** [comment]
@hannahburkhardt We just made a huge change to the codelab, and most of the text has changed. If any of the newer sections are still hard to read, please let us know.
